cpp/ql/lib/semmle/code/cpp/pointsto/PointsTo.qll

```diff
@@ -19,6 +19,9 @@
  * `pointstoinfo` predicate determines the transitively implied points-to
  * information by collapsing pointers into equivalence classes. These
  * equivalence classes are called "points-to sets".
+ *
+ * WARNING: This library may perform poorly on very large projects.
+ * Consider using another library such as ??? instead.
```

Comment:
Its difficult, because there isn't really a 1:1 replacement, but I should recommend *something* here.  `DataFlow`?  `GVN`?  Something else?

Reply:
I suggest `DataFlow`. `GVN` might also be a viable option in some cases, but I imagine `DataFlow` is the most common replacement.